/include/open62541/ua_architecture.h:145:10: fatal error: ua_architecture_functions.h: No such file or directory

Description
===========
When including <ua_types.h> I got the error:
[...]/include/open62541/ua_architecture.h:145:10: fatal error: ua_architecture_functions.h: No such file or directory
The file ua_architecture_function.h is not installed in [...]include/open62541/

@Pro When I get it right, the file should be added to these lines:
https://github.com/open62541/open62541/blob/20eccdcaf15162f48a372186e15f654d81331f59/CMakeLists.txt#L460-L464


Background Information / Reproduction Steps
===========================================
Install fresh on windows 10, include ua_types.h in any file.


Checklist
=========
Please provide the following information:

 - [x] open62541 Version (release number or git tag): 20eccdcaf15162f48a372186e15f654d81331f59
 - [ ] Other OPC UA SDKs used (client or server): 
 - [x] Operating system: Windows 10, Mingw-w64
 - [ ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ ] Wireshark network dump attached
 - [ ] Self-contained code example attached
 - [x] Critical issue

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1078,6 +1078,7 @@ if(NOT UA_ENABLE_AMALGAMATION)
                   ${exported_headers}
                   ${default_plugin_headers}
                   ${historizing_default_plugin_headers}
+                  ${ua_architecture_headers}
             DESTINATION ${open62541_install_include_dir})
 else()
     # Export amalgamated header open62541.h which is generated due to build of 

